templates/mail/wait-list.html

```diff
@@ -1,71 +1,109 @@
-            <input type="hidden" name="tags" value="10526643" />
-          </div>
-          <div id="mce-responses" class="clear">
-            <div
-              class="response"
-              id="mce-error-response"
-              style="display: none"
-            ></div>
-            <div
-              class="response"
-              id="mce-success-response"
-              style="display: none"
-            ></div>
-          </div>
-          <div style="position: absolute; left: -5000px" aria-hidden="true">
-            <input
-              type="text"
-              name="b_f8051cc8637cf3ff79661f382_a508104cf8"
-              tabindex="-1"
-              value=""
-            />
-          </div>
-          <div class="clear">
-            <input
-              type="submit"
-              name="subscribe"
-              id="mc-embedded-subscribe"
-              class="button"
-              value="Subscribe"
-            />
-          </div>
+{% load helpers %}
+{% load static %}
+
+
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title></title>
+  <link rel="stylesheet" href="{% static 'css/wait-list.css' %}">
+</head>
+<body>
+  <div class="container">
+    <header class="header">
+      <span>
+        <pre><code>$ thunderbird --version=pro</code></pre>
+      </span>
+      <h1>For Those Who Know</h1>
+      <h2>The power tool for digital communication</h2>
+    </header>
+    <section class="callout">
+      <span>
+        <code><pre>&gt;&gt; features</pre></code>
+      </span>
+      <span class="output">
+        <pre><code>[email_service, appointment, send, assist]</code></pre>
+      </span>
+    </section>
+    <section class="callout">
+      <span>
+        <pre><code>&gt;&gt; philosophy</code></pre>
+      </span>
+      <span class="output">
+        <pre><code>open_source & privacy_focused & user_controlled</code></pre>
+      </span>
+    </section>
+    <section class="callout">
+      <span>
+        <pre><code>&gt;&gt; status</code></pre>
+      </span>
+      <span class="output">
+        <pre><code>beta_signup.is_open=true</code></pre>
+      </span>
+    </section>
+    <div class="form-container">
+      <h3>Initialize Beta Access</h3>
+      <div id="mc_embed_shell">
+        <div id="mc_embed_signup">
+          <form
+            action="{{form_action}}"
+            method="post"
+            id="mc-embedded-subscribe-form"
+            name="mc-embedded-subscribe-form"
+            class="validate"
+            target="_self"
+            novalidate=""
+          >
+            <div id="mc_embed_signup_scroll">
+              <div class="mc-field-group form-inputs">
+                <input
+                  type="email"
+                  name="EMAIL"
+                  class="required email"
+                  id="mce-EMAIL"
```

Comment:
Is there a reason for uppercasing the name value?

Reply:
No "good" reason :laughing: 
That's copy/pasted from the MailChimp embed code. I decided not to bother cleaning it up.